src/main/java/net/mcreator/ui/dialogs/wysiwyg/ImageDialog.java

```diff
@@ -109,4 +121,8 @@ public ImageDialog(WYSIWYGEditor editor, @Nullable Image image) {
 		setVisible(true);
 	}
 
+	private boolean isAnimated(String image) {

```

Comment:
Given your feature does not depend on mcmeta, this is not the best assumption and one may eg. make fuel progress indicator which indeed does not need mcmeta given it is not a regular "animation".

A better check would be to check if height is a multiplier of width and in this case allow this parameter.

Reply:
✔️ Fixed